modular-docs-manual/files/TEMPLATE_REFERENCE_reference-material.adoc

```diff
@@ -20,15 +20,16 @@ The ID is an anchor that links to the module. Avoid changing it after the module
 [id="ref-reference-material_{context}"]
 
 ////
-The `context` attribute enables module reuse. Every module ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide
+The `context` attribute enables module reuse. Every module ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
 ////
 
 = Reference material
 ////
 In the title of a reference module, include nouns that are used in the body text. For example, "Keyboard shortcuts for ___" or "Command options for ___." This helps readers and search engines find the information quickly.
+
+Be sure to include a line break between the title and the module introduction.
```

Comment:
OT: Do we have an instruction for adding an empty line between every two includes somewhere? (this can prevent peculiar and hard-to-debug problems)

Reply:
Yes, in the assembly template.